src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java

```diff
@@ -529,7 +529,8 @@ private void createLine() throws LineUnavailableException, StreamPlayerException
 				outlet.setSourceDataLine((SourceDataLine) mixer.getLine(lineInfo));
 			}
 
-			outlet.setSourceDataLine((SourceDataLine) AudioSystem.getLine(lineInfo));
+			//Had to comment this because it overwrites the data line we just set, thus making code above useless (this line is already run when there is no mixer name set)
```

Comment:
I would just remove the line instead of deleting it. Maybe mention this in the commit message so that this note is tracked (instead of having the message in the code)